CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+<a name="1.15.10"></a>
+## <small>1.15.10 (2019-05-28)</small>
+
+* docs: update @bigcommerce/stencil-paper to 2.0.13
```

Comment:
Can you describe what paper 2.0.13 changes? And this should not be "docs" but rather "feat" (probably)

Reply:
fixed

Reply:
I put the number back down on package.json to 1.15.9.  Thinking 1.15.10 should encompass everything.

Reply:
I was told by Juned that there is a gulptask for generating changelog so deleted this.

Reply:
No changelog for cli it is auto generated on release. Plz make sure your commit message describes the change in detail.